### PR TITLE
Send capacity to multi sign script with since

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/models/multi-sign-utils.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    return {
+      codeHash,
+      outPoint,
+      hashType,
+    }
+  }
+
+  static async multiSignScript(nodeURL: string = NodeService.getInstance().ckb.rpc.node.url): Promise<SystemScript> {
+    if (MultiSignUtils.multiSignScriptInfo && nodeURL === MultiSignUtils.previousURL) {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/models/multi-sign-utils.ts]: Is a comparison between genesis block hashes better than that between URLs?

File: packages/neuron-wallet/src/models/multi-sign.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  hash(blake160: string): string {
+    return Blake2b.digest(this.serialize(blake160)).slice(0, 42)
+  }
+
+  since(minutes: number, headerEpoch: string): string {
+    const currentEpochInfo = this.parseEpoch(BigInt(headerEpoch))
+    const totalMinutes = minutes +
+      parseInt((parseInt(currentEpochInfo.index.toString()) / parseInt(currentEpochInfo.length.toString()) * 240).toString())
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/models/multi-sign.ts]: `240` is magical

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts [hunk 3]
```diff
@@ -32,18 +37,29 @@ export class TransactionGenerator {
...
     const outputs: Output[] = targetOutputs.map(o => {
-      const { capacity, address } = o
+      const { capacity, address, minutes } = o
 
-      const lockScript = new AddressParser(address)
+      let lockScript = new AddressParser(address)
         .setDefaultLockScript(codeHash, hashType)
         .parse()
 
+      if (minutes && minutes !== '') {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/tx/transaction-generator.ts]: `&& minutes !== ''` is redundant.

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts [hunk 4]
```diff
@@ -124,12 +142,21 @@ export class TransactionGenerator {
...
     const outputs: Output[] = targetOutputs.map((o, index) => {
-      const { capacity, address } = o
+      const { capacity, address, minutes } = o
 
-      const lockScript: Script = new AddressParser(address)
+      let lockScript: Script = new AddressParser(address)
         .setDefaultLockScript(codeHash, hashType)
         .parse()
 
+      if (minutes && minutes !== '') {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/tx/transaction-generator.ts]: Same as https://github.com/nervosnetwork/neuron/pull/1369/files#diff-101fa06357719ada7044c10dec1929f1R54

File: packages/neuron-wallet/src/models/multi-sign-utils.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/models/multi-sign-utils.ts]: Can't read from genesis block, especially off network.
And no need to update this alone, should refactor with `LockUtils` and `DaoUtils` and merge.

File: packages/neuron-wallet/src/models/multi-sign.ts (hunk 2 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/models/multi-sign.ts]: updated

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts (hunk 3 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx/transaction-generator.ts]: updated

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts (hunk 4 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/services/tx/transaction-generator.ts]: updated

**Pull request merged**